### ISSUE TITLE
Ads show up, but I get a message that they are blocked

### ISSUE THREAD

**Issue by jakobwildrain-si** [photopea/photopea#6944]
![image](https://github.com/photopea/photopea/assets/60407506/63fb4dcd-8508-4ede-864a-453b351d4ec1)

**Comment by jakobwildrain-si** [comment]
I'm using MS Edge without any extensions.
Ads show up in other websites too.

**Comment by camoody1** [comment]
Those are placeholder ads directly from the developer showing games he has created. He puts those there when the actual money-making ads are being blocked. You've got something enabled that is blocking the real ads.

**Comment by photopea** [comment]
Photopea is probably right, you are blocking ads. Did you try to follow the steps that you see in front of you?

**Comment by photopea** [comment]
I have to close it because of lack of details.

**Comment by photopea** [comment]
Also, what exactly in your screenshot do you call "ads"?

**Comment by jakobwildrain-si** [comment]
> Photopea is probably right, you are blocking ads. Did you try to follow the steps that you see in front of you?

Yes, but I haven't located any settings that block ads in MS Edge. In fact, I've enabled the setting that says Allow Intrusive ads. Same result.

No, I'm not using any ad blockers, never have. I'm a developer just like you, and I'm aware of what extensions I have installed.

Now that I look at the screenshot, I realize those are the ads directly from the dev, but they show up like regular ads, so without reading too much into those I would call them "ads". Wouldn't you?

I don't mind ads, I'm not trying to block them, I'm all in favor of supporting devs. I just don't know what else I can do specifically in MS Edge to allow ads. Any ideas?
